pkg/scaling/scale_handler.go

```diff
@@ -410,6 +410,8 @@ func buildScaler(name, namespace, triggerType string, resolvedEnv, triggerMetada
 		return scalers.NewRedisStreamsScaler(resolvedEnv, triggerMetadata, authParams)
 	case "stan":
 		return scalers.NewStanScaler(resolvedEnv, triggerMetadata)
+	case "external-metric-source":
```

Comment:
Scalers are alphabetically sorted in the switch, please move it to the correct location :)

Reply:
Will do once we agree on the name and I will add check for this in #1025 :)